pkgs/development/tools/electron/common.nix

```diff
@@ -247,6 +254,12 @@ in
     mkdir -p $libExecPath
     unzip -d $libExecPath out/Release/dist.zip
 
+    # Create reproducible tarball, per instructions at https://reproducible-builds.org/docs/archives/
+    tar --sort=name \
+      --mtime="@$SOURCE_DATE_EPOCH" \
+      --owner=0 --group=0 --numeric-owner \
```

Comment:
For reference, over at chromium: https://github.com/NixOS/nixpkgs/blob/85795c79e95877dafcb1a202d275b09b84a20f4e/pkgs/applications/networking/browsers/chromium/common.nix#L228-L237

I think `reproducible-builds.org` made changes to their suggestions again, good to know.

Using `--owner=0` instead of `--owner=root` sounds to me, even if it resolves to effectively the same due thanks to Nix and its sandbox.

Likewise, using `--mtime="@$SOURCE_DATE_EPOCH"` instead of `--mtime="1970-01-01"` solves the timezone ambiguity (without using `"1970-01-01Z"`), if Nix wouldn't handle that for us already.

I think I will apply those changes to our chromium tar generation, assuming I won't forget :)